### ISSUE TITLE
List all deployed document store

### ISSUE THREAD

**Issue by Nebulis** [OpenCerts/admin-website-v2#34]
Is it possible ?

**Comment by ChenRen93** [comment]
This is possible. I have just created this function, I am thinking of changing the document store textbox to a combo box, then in addition I can validate the document-store using this as well.

**Comment by Nebulis** [comment]
Combo box can. Possible to display the name of the doc store?

**Comment by ChenRen93** [comment]
yes, i can use combobox/datalist to display all the document store created by the active wallet.

**Comment by Nebulis** [comment]
Let's go then

**Comment by ChenRen93** [comment]
Dated : 24 August 2021

Method tried :
getLogs() is able to find the deployed document store. however, this method is limited to retrieving the document store contracts that are of the ^0.5.12 version. it will an issue to retrieval of a contract of the older version and will cause an issue if there is an upgrade on the upcoming/future contracts refactoring.

Dated at this point, there is no library that can directly retrieve the deployed document store or any methods to retrieve without any complication. This feature will be ice-box till DLT libraries are advanced or further revisit in future.

**Comment by Nebulis** [comment]
let's keep it open :)
